qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -1835,7 +1835,6 @@ def __json_encode__(self):
         for att in [
             "_metadata",
             "_source",
-            "_service",
```

Comment:
`_backend` should probably be removed from here too since it isn't properly serializable.

Reply:
changed the code and added the following functionality:
```python
for attr in ["_service", "_backend"]:
	json_value[attr] = None
	value = getattr(self, attr)
	if value is not None:
		LOG.info("%s cannot be JSON serialized", str(type(value)))
```
Resolving this comment.